miso-service/src/test/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultExperimentServiceTestSuit.java

```diff
@@ -0,0 +1,133 @@
+package uk.ac.bbsrc.tgac.miso.service.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.*;
+
+import java.io.IOException;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.invocation.InvocationOnMock;
+import org.mockito.stubbing.Answer;
+
+import com.eaglegenomics.simlims.core.User;
+
+import uk.ac.bbsrc.tgac.miso.core.data.Experiment;
+import uk.ac.bbsrc.tgac.miso.core.security.SecurableByProfile;
+import uk.ac.bbsrc.tgac.miso.core.store.ExperimentStore;
+import uk.ac.bbsrc.tgac.miso.core.store.KitStore;
+import uk.ac.bbsrc.tgac.miso.core.store.PlatformStore;
+import uk.ac.bbsrc.tgac.miso.core.store.PoolStore;
+import uk.ac.bbsrc.tgac.miso.core.store.SecurityStore;
+import uk.ac.bbsrc.tgac.miso.core.store.StudyStore;
+import uk.ac.bbsrc.tgac.miso.service.security.AuthorizationManager;
+
+public class DefaultExperimentServiceTestSuit {
+
+  @Rule
+  public final ExpectedException thrown = ExpectedException.none();
+
+  @Mock
+  private Experiment experiment;
+
+  @Mock
+  private ExperimentStore experimentStore;
+  @Mock
+  private PlatformStore platformStore;
+  @Mock
+  private PoolStore poolStore;
+  @Mock
+  private StudyStore studyStore;
+  @Mock
+  private SecurityStore securityStore;
+  @Mock
+  private KitStore kitStore;
+  @Mock
+  private AuthorizationManager authorizationManager;
+  @InjectMocks
+  private DefaultExperimentService sut;
+
+  @Before
+  public void setUp() {
+    MockitoAnnotations.initMocks(this);
+  }
+
+  /**
+   * Test method for
+   * {@link uk.ac.bbsrc.tgac.miso.core.manager.UserAuthMisoRequestManager#saveExperiment(uk.ac.bbsrc.tgac.miso.core.data.Experiment)} .
+   */
+  @Test
+  public void testSaveExperiment() throws IOException {
+    final long expectedReturn = 1L;
+    doAnswer(new Answer<Void>() {
```

Comment:
Was this necessary to make the mock do nothing when a void-return method is called? I thought that was the default behaviour